packages/bump-monorepo-packages/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@mongodb-js/bump-monorepo-packages",
+  "description": "Compute and writes conventional bumps for monorepo packages",
+  "author": {
+    "name": "MongoDB Inc",
+    "email": "compass@mongodb.com"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "bin": {
+    "bump-monorepo-packages": "./src/bump-packages.js"
+  },
+  "bugs": {
+    "url": "https://jira.mongodb.org/projects/COMPASS/issues",
+    "email": "compass@mongodb.com"
+  },
+  "homepage": "https://github.com/mongodb-js/compass",
+  "version": "0.1.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mongodb-js/compass.git"
+  },
+  "files": [
+    "dist"
+  ],
+  "license": "SSPL",
+  "main": "dist/index.js",
+  "compass:main": "src/index.ts",
```

Comment:
These are webpack specific keys used in compass that we probably don't need here